lib/ignore/ignore.go

```diff
@@ -55,26 +69,26 @@ func New(withCache bool) *Matcher {
 	return m
 }
 
-func (m *Matcher) Load(file string) error {
+func (m *Matcher) Load(file string, includePath string) error {
```

Comment:
I don't think so. It gets passed all the way down, but in the end we just

```
includeFile := filepath.Join(filepath.Dir(currentFile), includeRel)
```

so I don't think it's used?